beacon-chain/db/kv/state.go

```diff
@@ -401,3 +613,22 @@ func (s *Store) CleanUpDirtyStates(ctx context.Context, slotsPerArchivedPoint ty
 
 	return err
 }
+
+func (s *Store) isStateValidatorMigrationOver() bool {
+	// if flag is enabled, then always follow the new code path.
+	if featureconfig.Get().EnableHistoricalSpaceRepresentation {
+		return true
+	} else {
```

Comment:
```suggestion
	}
```
No need for the if - else here, as there is only one path to follow below